src/mantle/framework/class-alias-loader.php

```diff
@@ -72,6 +72,8 @@ public function load( $alias ) {
 		if ( isset( $this->aliases[ $alias ] ) ) {
 			return class_alias( $this->aliases[ $alias ], $alias );
 		}
+
+		return null;
```

Comment:
The callback's signature is ` callback(string $class): void`
https://www.php.net/manual/en/function.spl-autoload-register.php

`spl_autoload_register` itself returns a boolean.

Reply:
Then you can remove `@phpstan-ignore-line` 💝